app/views/clients/show.html.erb

```diff
@@ -41,7 +27,8 @@
   <%end%>
 
       <%= react_component("Identity", props: {id: @client.nil? ? '' : @client.id,
-        identities: @identities
+        identities: @identities,
+        country: @country_of_birth
```

Comment:
why pass this? It's already there on the `@identities`

Every identity record in the `@identities` array will have a `country_of_birth` and an `issuing state`.